editor/editor_about.cpp

```diff
@@ -73,15 +87,22 @@ void EditorAbout::_version_button_pressed() {
 	DisplayServer::get_singleton()->clipboard_set(version_btn->get_meta(META_TEXT_TO_COPY));
 }
 
-void EditorAbout::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("_version_button_pressed"), &EditorAbout::_version_button_pressed);
+void EditorAbout::_item_with_website_selected(int p_id, ItemList *p_il) {
+	const String website = String(p_il->get_item_metadata(p_id));
+	if (!website.is_empty()) {
+		OS::get_singleton()->shell_open(website);
+	}
 }
 
-TextureRect *EditorAbout::get_logo() const {
-	return _logo;
+void EditorAbout::_item_list_resized(ItemList *p_il) {
+	p_il->set_fixed_column_width(p_il->get_size().x / 3.0 - 16 * EDSCALE * 2.5); // Weird. Should be 3.0 and that's it?.
```

Comment:
Is this supposed to fit the columns to the whole list? Because it doesn't
![image](https://github.com/godotengine/godot/assets/2223172/da84eb22-72de-43c6-ba43-21bc1a9b9ada)
I think this should be solved in ItemList with something like `expand_columns`.

Reply:
I tried really hard to, but for some reason any value higher than the formula in the current PR causes the list to use two columns instead.

Reply:
How does it look without this code?
This is why I said it should be solved by ItemList, which can be done later if it looks *ok* without the size adjustment.

Reply:
You mean, just by dividing by 3.0? I can show that in a moment. It displays 2 columns only.

Reply:
I mean without fixed column width at all.

Reply:
Misaligned :/
So I guess this code stays.